examples/binary-bitwise-expressions/binary_bitwise_expressions.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/io;
+import ballerina/lang.'int as ints;
```

Comment:
Thought we weren't using aliases for lang libs in examples?

Reply:
We weren't?

Did a quick search for "import ballerina/lang" and we seem to be using aliases in all the examples?

Reply:
Removed the alias as discussed.